toolchain/ltrace.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+
+import os
+from avocado import Test
+from avocado import main
+from avocado.utils import build
+from avocado.utils import process
+from avocado.utils import git
+from avocado.utils import distro
+from avocado.utils.software_manager import SoftwareManager
+
+backend = SoftwareManager()
+dist = distro.detect()
```

Comment:
Top level variables should use UPPER_CASE only. But I see you use it only inside the `setUp` method, would you please move it there, instead to avoid import exceptions and unnecessary memory overhead?